### PR TITLE
docs: Start changelog for 7.9.0

### PR DESCRIPTION
**Pull request by kuisathaverat**
initial Changelog for 7.9.0

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -3,6 +3,880 @@
...
+* Checkpoint {pull}18754[18754]
+* Netflow {pull}19087[19087]
+* Zeek {pull}19113[19113] (`forwarded` tag is not included by default)
+* Suricata {pull}19107[19107] (`forwarded` tag is not included by default)
+* CoreDNS {pull}19134[19134] (`forwarded` tag is not included by default)
+* Envoy Proxy {pull}19134[19134] (`forwarded` tag is not included by default)
+- With the default configuration the cloud modules (aws, azure, googlecloud, o365, okta)
+will no longer send the `host` field that contains information about the host Filebeat is
+running on. This is because the `host` field specifies the host on which the event
+happened. {issue}13920[13920] {pull}18223[18223]
```
Comment (urso) [inline CHANGELOG.asciidoc:40]: duplicate entry.

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -3,6 +3,880 @@
...
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix potential race condition in fingerprint processor. {pull}18738[18738]
+- Add better handling for Kubernetes Update and Delete watcher events. {pull}18882[18882]
+- Fix config reload metrics (`libbeat.config.module.start/stops/running`). {pull}19168[19168]
+- Fix metrics hints builder to avoid wrong container metadata usage when port is not exposed {pull}18979[18979]
+- Server-side TLS config now validates certificate and key are both specified {pull}19584[19584]
```
Comment (urso) [inline CHANGELOG.asciidoc:88]: Maybe better: `Server-side TLS config now validates that certificate and key settings are both specified`

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -3,6 +3,880 @@
...
+
+
+*Functionbeat*
+
+
+==== Added
+
+*Affecting all Beats*
+
+- Add configuration for APM instrumentation and expose the tracer trough the Beat object. {pull}17938[17938]
```
Comment (urso) [inline CHANGELOG.asciidoc:137]: No need to include technical implementation details in user facing changelog. Maybe better: `- Add initial instrument of Beats with APM GO Agent`.

File: CHANGELOG.asciidoc [hunk 4]
```diff
@@ -3,6 +3,880 @@
...
+
+*Heartbeat*
+
+- Record HTTP response headers. {pull}18327[18327]
+
+*Heartbeat*
+
+*Journalbeat*
+
+- Added an `id` config option to inputs to allow running multiple inputs on the
```
Comment (urso) [inline CHANGELOG.asciidoc:207]: ?

Comment (kuisathaverat) [inline CHANGELOG.asciidoc:207]:
```suggestion
- Added an `id` config option to inputs to allow running multiple inputs on the same journal. {pull}18467{18467}
```

File: CHANGELOG.asciidoc (hunk 3 above)
Comment (kuisathaverat) [inline CHANGELOG.asciidoc:137]:
```suggestion
- Add initial instrument of Beats with APM GO Agent. {pull}17938[17938]
```

File: CHANGELOG.asciidoc (hunk 2 above)
Comment (kuisathaverat) [reply CHANGELOG.asciidoc:88]:
```suggestion
- Server-side TLS config now validates that certificate and key settings are both specified {pull}19584[19584]
```

File: CHANGELOG.asciidoc [hunk 5]
```diff
@@ -3,6 +3,880 @@
...
+* CrowdStrike {pull}19132[19132]
+* Fortinet {pull}19133[19133]
+* iptables {pull}18756[18756]
+* Checkpoint {pull}18754[18754]
+* Netflow {pull}19087[19087]
+* Zeek {pull}19113[19113] (`forwarded` tag is not included by default)
+* Suricata {pull}19107[19107] (`forwarded` tag is not included by default)
+* CoreDNS {pull}19134[19134] (`forwarded` tag is not included by default)
+* Envoy Proxy {pull}19134[19134] (`forwarded` tag is not included by default)
+- With the default configuration the cloud modules (aws, azure, googlecloud, o365, okta)
```
Comment (kuisathaverat) [inline CHANGELOG.asciidoc:37]:
```suggestion
```

File: CHANGELOG.asciidoc [hunk 6]
```diff
@@ -3,6 +3,880 @@
...
+will no longer send the `host` field that contains information about the host Filebeat is
```
Comment (kuisathaverat) [inline CHANGELOG.asciidoc:38]:
```suggestion
```

File: CHANGELOG.asciidoc [hunk 7]
```diff
@@ -3,6 +3,880 @@
...
+running on. This is because the `host` field specifies the host on which the event
```
Comment (kuisathaverat) [inline CHANGELOG.asciidoc:39]:
```suggestion
```

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (kuisathaverat) [inline CHANGELOG.asciidoc:40]:
```suggestion
```

Comment (kuisathaverat) [reply CHANGELOG.asciidoc]: removed

File: CHANGELOG.asciidoc (hunk 3 above)
Comment (kuisathaverat) [reply CHANGELOG.asciidoc]: changed

File: CHANGELOG.asciidoc (hunk 4 above)
Comment (kuisathaverat) [reply CHANGELOG.asciidoc]: fixed

**Pull request merged**